### PR TITLE
Obsolete paymentType

### PR DESCRIPTION
**Pull request by ondrahermanek**
image

### CODE REVIEW

**Comment by MikeAdamsMews** [comment]
@ondrahermanek from your description, the `PaymentType` field inside `RatePlan` is effectively deprecated, is that right? I mean, the field isn't going away anytime soon but you're saying don't use it because from this point onwards the information won't be reliable. That sounds like a deprecation to me. Bear in mind that I don't know the details of how rate plans or settlement plans work, so I'm just looking at it from a surface point of view.

Assuming it is a deprecation, then we should follow the standard process. We use  ~~strikethrough~~ and mark it with the keyword **Deprecated!** and then add an entry to the **Deprecations Table**. That should include what action the user should take as an alternative, in this case where might they go to get the `Payment Type`?

I do think the explanation paragraph you've written would be helpful to users, the only question is where to put it. I want to avoid putting lengthy explanations inside the API Reference, especially looking to the future when we hope to automate generating the API Reference from an OpenAPI definition. I wonder if it would be better to put in on the Deprecations page, because it is directly concerned with the deprecation. What do you think?

**Comment by ondrahermanek** [comment]
@MikeAdamsMews Yep, make sense. I changed the PR. I moved the note to the deprecations. Also I changed the end a bit, so if you could double check if it still makes sense and is correct.

File: mews-operations/configuration.md [hunk 1]
```diff
@@ -423,7 +421,7 @@ This is an example of a _successful_ response. In case an error occurred, the re
 | `name` | `string` | required | Name of the rate plan. |
 | `currencyCode` | `string` | required | Three-letter currency code of the rate plan price. |
 | `description` | `string` | optional | Description of the rate plan. |
-| `paymentType` | `int` | required | [`Payment type`](#payment-types) code. |
+| ~~`paymentType`~~ | ~~`int`~~ | ~~required~~ | ~~[`Payment type`](#payment-types) code.~~ [**Deprecated**](../deprecations/README.md)|
```
Comment (MikeAdamsMews) [inline mews-operations/configuration.md:424]:
```suggestion
| ~~`paymentType`~~ | ~~`int`~~ | ~~required~~ | ~~[`Payment type`](#payment-types) code.~~ [**Deprecated!**](../deprecations/README.md)|
```

**Review by MikeAdamsMews — APPROVED** [review]
Thanks @ondrahermanek, that's better. I moved the entry in the Deprecations table to the top, because the newest ones are at the top of the table.

**Comment by ondrahermanek** [comment]
Makes sense, thanks @MikeAdamsMews !

**Pull request merged**